intrinsics_tester/compareFunctions.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright (c) 2018-present The Alive2 Authors.
```

Comment:
I guess this file is copy-pasted from tools/transform.cpp. Can we reuse the compareFunction from tools/transform.cpp instead of calling this copy?